libr/bin/bfile.c

```diff
@@ -34,9 +35,21 @@ static void print_string(RBinFile *bf, RBinString *string) {
 	type_string = r_bin_string_type (string->type);
 	vaddr = addr = r_bin_get_vaddr (bin, string->paddr, string->vaddr);
 
+	// If raw string dump mode, use printf to dump directly to stdout.
+	PrintfCallback temp = io->cb_printf;
+	if (raw == 2) {
+		io->cb_printf = r_cons_printf;
```

Comment:
you can only do this from core, rcons is not visible by rbin

Reply:
Previously you told me to use r_cons_printf (and not printf)

https://github.com/radare/radare2/pull/12578#discussion_r244532220

I will try to find another print function